internal/budgetadjustments/flags.go

```diff
@@ -0,0 +1,96 @@
+package budgetadjustments
+
+import (
+	"time"
+
+	"github.com/spf13/cobra"
+
+	"github.com/nobl9/sloctl/internal/csv"
+)
+
+const (
+	flagAdjustment = "adjustment-name"
```

Comment:
maybe just name? we have `sloctl budgetadjustments --adjustment-name` for me adjustment in flag is uncessary duplication

Reply:
We are getting events by adjustment name so lets leave it as it is